server/controllers/notificationController.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable require-jsdoc */
+import env from 'dotenv';
+import Model from '../models';
+import notificationMessages from '../helpers/notificationMessages';
+import sendMail from '../helpers/mail';
+
+/* eslint-disable no-unused-vars */
+const { User, Followings, Notification } = Model;
+
+
+env.config();
+
+class Notifications {
```

Comment:
This method finds all the followers of a particular user and then notifies them about a new article the user added.